packages/editor-ui/src/hooks/posthog/log.ts

```diff
@@ -0,0 +1,16 @@
+import { n8nPosthogHooks_LOGGING_ENABLED } from '@/hooks/constants';
+
+export function hooksPosthogLog(
+	name: string,
+	{ isMethod } = { isMethod: false },
+	loggingEnabled = n8nPosthogHooks_LOGGING_ENABLED,
+) {
+	if (!loggingEnabled) return;
+
+	if (isMethod) {
+		console.log(`Method fired: ${name}`);
+		return;
+	}
+
+	console.log(`Hook fired: ${name}`);
```

Comment:
not sure we need these.. have posthog debug mode setup in the new posthog store

Reply:
Agreed. Removed the logging